pkg/products/marin3r/config/config.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 const (
-	RateLimitConfigMapName = "sku-limits-managed-api-service"
+	RateLimitConfigMapName = "quota-limits-managed-api-service"
```

Comment:
@jackdelahunt we need to leave this one the same as it refers to a preexisiting config map -> https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/main/addons/managed-api-service/metadata/stage/sku-limits.yaml.j2